README.md

```diff
@@ -182,7 +182,7 @@ have found a better experience using XWayland which can be achieved launching
 Alacritty with the `WAYLAND_DISPLAY` environment variable cleared:
 
 ```sh
-env WAYLAND_DISPLAY= alacritty
+env WAYLAND_DISPLAY=alacritty
```

Comment:
Above section says "which can be achieved launching Alacritty with the `WAYLAND_DISPLAY` environment variable cleared", which makes me believe this change (_setting_ the `WAYLAND_DISPLAY` env var as opposed to _clearing_ it) was just a brainfart. If this change is intentional, maybe the above should be reworded.

Also in that same section, maybe "which can be achieved launching Alacritty" -> "which can be achieved **by** launching Alacritty"

Reply:
@cole-h much thanks for catching this. I've updated the PR to hopefully be a bit more clear, since I honestly forgot `env A= ...` is clearing `A` (not technically unsetting). An empty string should make this more obvious hopefully.